index.js

```diff
@@ -74,6 +78,8 @@ function togpx( geojson, options ) {
   else
     features = [{type:"Feature", properties: {}, geometry: geojson}];
   features.forEach(function mapFeature(f) {
+    if (!f.hasOwnProperty('properties'))
+      f.properties = {};
```

Comment:
I've replaced this now with more robust versions of the default `get_*` callbacks that just return nothing in the case of a missing feature properties object.